src/lib/learn-client/index.ts

```diff
@@ -2,19 +2,6 @@
  * Copyright (c) HashiCorp, Inc.
  * SPDX-License-Identifier: MPL-2.0
  */
-function getFetch() {
```

Comment:
This was removed in favor of native `fetch` to (hopefully) improve accuracy of Vercel's "Request Metrics" feature.